### PR TITLE
Playwright: additional unit tests for calypso-e2e module.

### LINKED ISSUE #53033: [Meta] Playwright: add unit tests for @automattic/calypso-e2e module
**Task Details**
Foundational work for Playwright migration is being put in the @automattic/calypso-e2e package.
However, helper functions that interact with the system, config values and environment variables do not have tests written for them. 

This makes guaranteeing the output of these functions more difficult.

When filing a PR for writing additional tests of the `@automattic/calypso-e2e` module, make sure to use this issue as the parent issue.

**Related to**
#51082

### LINKED ISSUE #53644: Playwright: merge getScreenshotName and getVideoName in MediaHelper.
**Task Details**
These files do basically the same thing, so let's merge it and write tests.

**Related to**
#53513

### PR DESCRIPTION
**Pull request by worldomonation**
#### Changes proposed in this Pull Request

This PR implements additional unit tests for code I have written in `calypso-e2e` module.

Furthermore, there have been some refactoring of the helper files:
- `DataHelper.toTitleCase`, to no longer lowercase characters from 2nd char onwards.
- `getScreenshotName` and `getVideoName` have been refactored into one function.

#### Testing instructions

Local
- run `yarn jest` and ensure all tests pass.

TeamCity
- ensure all tests pass for both `Unit Tests` and `Playwright E2E`.
*note*: E2E failure are due to this being based on old trunk.

### CODE REVIEW

File: packages/calypso-e2e/src/data-helper.ts
Comment (worldomonation) [inline packages/calypso-e2e/src/data-helper.ts:96]: No longer lowercases non-head characters.

As a result, with the input `SEO (Manage)`, this function:

`Seo (manage)` --> `SEO (Manage)`

**Review by bsessions85 — APPROVED** [review]
Looks good. Nice update to the media helper.

**Comment by worldomonation** [comment]
Ready to merge, pending further approval and review of the parent PR #53291.

**Pull request merged**
